GROUPS.md

```diff
@@ -16,7 +16,7 @@ you'll have access to all the docs and won't need to request access for each of
 
 | Name | Leads | Mailing List | Example Topics | Meetings |
 |------|-------|---------|----------|----------|
-| Core | Sven Mawson (Google), Louis Ryan (Google), Martin Taillefer (Google), Shriram Rajagopalan (IBM), Dan Berg (IBM), Lin Sun(IBM) | [istio-core@](https://groups.google.com/forum/#!forum/istio-core) | Configuration, Performance, Stability | TBD
+| Core | Sven Mawson (Google), Louis Ryan (Google), Martin Taillefer (Google), Shriram Rajagopalan (IBM), Dan Berg (IBM), Lin Sun(IBM) | [istio-core@](https://groups.google.com/forum/#!forum/istio-core) | Configuration, Performance, Stability | [Mondays at 13:00 PT (weekly)](https://groups.google.com/forum/#!forum/istio-0-3-progr)
```

Comment:
I don't think that's the right meeting link :-)

Should we add the config working group info? Sven never got around to doing that.

Reply:
That was very sloppy of me - fixed the incorrect link. 
About config, Joy is polling the date/time/frequency for meetings, we could wait until that settles before adding a row - wdyt?